### PR TITLE
Extend the time frame for the crash bug summary

### LINKED ISSUE #2155: Extend the time frame that is used to generate the crash bug summary

### PR DESCRIPTION
**Pull request by suhaibmujahid**
Resolves #2155 

## Checklist

- [ ] Type annotations added to new functions
- [ ] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results
- [ ] The [`to-be-announced`](https://github.com/mozilla/bugbot/labels/to-be-announced) tag added if this is worth announcing

### CODE REVIEW

**Comment by suhaibmujahid** [comment]
# Example from the dry-run

Crash report: https://crash-stats.mozilla.org/report/index/a1baa457-5a02-419c-94d5-cce680230624



MOZ_CRASH Reason: ```MOZ_DIAGNOSTIC_ASSERT(0)```


Top 10 frames of crashing thread:
```
0  firefox-bin  arena_t::ArenaRunRegAlloc  memory/build/mozjemalloc.cpp:2504
0  firefox-bin  arena_t::MallocSmall  memory/build/mozjemalloc.cpp:3235
0  firefox-bin  arena_t::Malloc  memory/build/mozjemalloc.cpp:3281
0  firefox-bin  BaseAllocator::malloc  memory/build/mozjemalloc.cpp:4485
0  firefox-bin  Allocator<MozJemallocBase>::malloc  memory/build/malloc_decls.h:51
0  firefox-bin  PageMalloc  memory/replace/phc/PHC.cpp:1197
0  firefox-bin  replace_malloc  memory/replace/phc/PHC.cpp:1201
0  firefox-bin  Allocator<ReplaceMallocBase>::malloc  memory/build/malloc_decls.h:51
0  firefox-bin  malloc  memory/build/malloc_decls.h:51
1  libxul.so  nsStringBuffer::Alloc  xpcom/string/nsStringBuffer.cpp:68
```



Here are some insights about the crash signature based on the last 2 months of Nightly data:

- **Crash kind:** Browser Crash
- **Is startup crash:** No
- **Has user comments:** No
- **Is null crash:** Yes - 4 out of 6 crashes happened on null or near null memory address
- **Is use after free crash:** No




By analyzing the backtrace, the regression may have been introduced by a patch [1] to fix [Bug 1836879](https://bugzilla.mozilla.org/show_bug.cgi?id=1836879).

[1] https://hg.mozilla.org/mozilla-central/rev?node=2e153261c79c



:bwc, since you are the author of the potential regressor, could you please take a look?

File: templates/file_crash_bug_description.md.jinja [hunk 1]
```diff
@@ -28,7 +28,7 @@ Top {{ frames|length }} frames of crashing thread:
 {% endif %}
 
 
-Here are some insights about the crash signature based on recent data at the time of reporting the bug:
+Here are some insights about the crash signature based on the last {{ signature_details_delta }} of {{ signature_details_channel }} data:
```
Comment (marco-c) [inline templates/file_crash_bug_description.md.jinja:31]: When the signature is new, e.g. just started 2 days ago, it'd be confusing to say "based on the last 10 months of ...". Could we check when the signature started and replace the "10 months" with that if it's later than 10 months ago?

Comment (suhaibmujahid) [reply templates/file_crash_bug_description.md.jinja:31]: I agree it can be confusing. I do not feel changing the number will make it clearer. Maybe rewording can make things less confusing. What do you think about the following rewarding?


> By querying Nigtly crashes reported within the last 2 months, here are some insights about the signature:

We could even make it more explicit by adding a list item about the start date: 

> - First crash report: 2023-07-10

Comment (marco-c) [reply templates/file_crash_bug_description.md.jinja:31]: Rewording and making it more explicit sounds good to me!

**Review by marco-c — APPROVED** [review]
